virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,71 +1,40 @@
 :_content-type: ASSEMBLY
-[id="attaching-to-multiple-networks"]
-= Attaching a virtual machine to multiple networks
+[id="virt-attaching-vm-multiple-networks"]
+= Attaching a virtual machine to an external network
 include::_attributes/common-attributes.adoc[]
-:context: virt-attaching-multiple-networks
+:context: virt-attaching-vm-multiple-networks
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+By default, {VirtProductName} is installed with a single, internal pod network.
 
-To attach a virtual machine to an additional network:
+You must create a Linux bridge network attachment definition (NAD) in order to connect to external networks. Otherwise, your network interface cards will be unavailable.
 
-. Configure a Linux bridge.
-. Configure a bridge network attachment definition for a namespace in the web console or CLI.
-+
-[NOTE]
-====
-The network attachment definition must be in the same namespace as the pod or virtual machine.
-====
-. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
-** In the web console, create a NIC for a new or existing virtual machine.
-** In the CLI, include the network information in the virtual machine configuration.
-
-[NOTE]
-====
-There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
-However, a network attachment definition provides a more efficient and more manageable configuration.
-====
-
-include::modules/virt-networking-glossary.adoc[leveloffset=+1]
+To attach a virtual machine to an external network:
 
-[id="virt-creating-linux-bridge"]
-== Configuring a Linux bridge
+. Create a `NodeNetworkConfigurationPolicy`.
```

Comment:
I think this might be clearer:
"Create a Linux bridge node network configuration policy."

Reply:
Maybe _Create a Linux bridge network configuration policy_ is more consistent? Node is implicit and conflicts with other instances of _Linux bridge network_

Reply:
Going with _Create a Linux bridge network configuration policy_ unless you have objections.

Reply:
> Maybe _Create a Linux bridge network configuration policy_ is more consistent? Node is implicit and conflicts with other instances of _Linux bridge network_

No, consistency is not the goal here because we are talking about different contexts:

- VM: Linux bridge network interface
- cluster network: Linux bridge network attachment definition
- cluster node: Linux bridge node network configuration policy

VM, network, and node are all different entities. It's good to be consistent but not at the expense of clarity.